<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebViewPropertyGridWidget</class>
 <widget class="QWidget" name="WebViewPropertyGridWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>120</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>299</width>
     <height>119</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>20</y>
      <width>290</width>
      <height>95</height>
     </rect>
    </property>
    <property name="title">
     <string>Data detectors</string>
    </property>
    <widget class="QCheckBox" name="phoneNumbersCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>121</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Phone numbers</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="calendarEventsCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>121</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Calendar events</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="linksCheckbox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>85</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Links</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="addressesCheckbox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>85</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Addresses</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="noneCheckbox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>85</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="allCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>85</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>All</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>allCheckbox</tabstop>
  <tabstop>phoneNumbersCheckbox</tabstop>
  <tabstop>calendarEventsCheckbox</tabstop>
  <tabstop>noneCheckbox</tabstop>
  <tabstop>linksCheckbox</tabstop>
  <tabstop>addressesCheckbox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
